product-service/handlers/getProductsById.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import { mockAsyncCall } from '../mocks/async.mock';
+import { productsMockData } from '../mocks/products.mock';
+
+export const getProductsById = async (event) => {
+    console.log('[Get Products List By ID] request, event:', event);
+
+    const headers = {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Credentials': true,
+    };
+
+    const productsData = await mockAsyncCall(productsMockData);
+    let productId = null;
+
+    if(event && event.pathParameters && event.pathParameters.productId != null){
+        productId = event.pathParameters.productId;
+    } else{
+        return {
```

Comment:
You can create some util function to return response passing proper params. this way you will avoid code duplication and won't create headers for every function
It is applicable for both for products and productId, success and error responses

Reply:
Created separate function as suggested